src/hooks/useOffersLink.ts

```diff
@@ -20,14 +20,20 @@ export const useOffersLink = (): undefined | string | null => {
   useEffect(() => {
     const asyncCore = async (): Promise<void> => {
       try {
-        const { data: context } = (await client.query(
-          Q('io.cozy.settings').getById('context')
-        )) as ExpectedContext
-
-        const { data: instance } = (await client.query(
-          Q('io.cozy.settings').getById('instance')
-        )) as ExpectedInstance
-
+        const [{ data: context }, { data: instance }] = await Promise.all([
+          (await client.fetchQueryAndGetFromState({
+            definition: Q('io.cozy.settings').getById('context'),
+            options: {
+              fetchPolicy: CozyClient.fetchPolicies.olderThan(30 * 100000)
```

Comment:
We often read in our codebase `olderThan(30 * 1000)` which is easily understandable as `30 x 1s` because we are used to manipulate milliseconds.

Here you used `30 * 100000` which may be confusing. A reader may wonder what is the unit here (I was going to give microsecond as exemple but then I noticed there were only 5 zeros 😅 )

I would prefer `3000 * 1000` or better : a named constant

Reply:
I would prefer https://github.com/cozy/cozy-client/issues/1253 :)

Reply:
(force pushed and changed to `3000 * 1000`)